Framework/Algorithms/src/SofQWNormalisedPolygon.cpp

```diff
@@ -177,8 +400,8 @@ void SofQWNormalisedPolygon::exec() {
       const V2D ur(dE_jp1, m_EmodeProperties.q(dE_jp1, thetaUpper, det));
       const V2D ul(dE_j, m_EmodeProperties.q(dE_j, thetaUpper, det));
       if (g_log.is(Logger::Priority::PRIO_DEBUG)) {
-        logStream << "Spectrum=" << specNo << ", theta=" << theta
-                  << ",thetaWidth=" << thetaWidth << ". QE polygon: ll=" << ll
+        logStream << "Spectrum=" << specNo << ", lower theta=" << thetaLower
```

Comment:
Is there any particular reason to output these angles in radians? If not, I'd recommend to output them in degrees to be consistent.